src/trace_processor/metrics/sql/android/jank/relevant_threads.sql

```diff
@@ -57,6 +57,14 @@ DROP TABLE IF EXISTS android_jank_cuj_sf_process;
 CREATE PERFETTO TABLE android_jank_cuj_sf_process AS
 SELECT * FROM _android_sf_process;
 
+DROP TABLE IF EXISTS android_jank_cuj_sf_main_thread;
```

Comment:
Please leave TODOs that these only exist for Google3 backcompat and should be deleted when we migrate the pipelines away from using them.